website/docs/docs/build/packages.md

```diff
@@ -173,19 +173,37 @@ To use native private packages, you must have one of the following Git providers
 
 #### Configuration
 
-Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable:
+Use the `private` key in your `packages.yml` or `dependencies.yml` to clone package repos using your existing dbt Cloud Git integration without having to provision an access token or create a dbt Cloud environment variable. 
+
+The private key supports a two-layered-path: `org/repo`. dbt inherits the project path from your source repository.
 
 <File name="packages.yml">
 
 ```yaml
 packages:
-  - private: dbt-labs/awesome_repo
+  - private: dbt-labs/awesome_repo # your-org/your-repo path
   - package: normal packages
-
-	[...]
+  [...]
 ```
+</File>
+
+:::tip Azure DevOps repositories
+Normally ADO repos use the `org_name/project_name/repo_name` path. However, private package supports only the `org/repo` path. If the ADO package repository is within the same project as your source repository, you can specify `org/repo` in the `private` key. If the package repository _isn't_ within the same project as your source repository, then the private packages feature won't currently work:
```

Comment:
I recommend starting with the take away from  this callout. Maybe we can also simplify it a little?

```suggestion
Currently, the private package feature only works when the ADO package repository is within the same project as your source repository. Normally ADO repos use the `org_name/project_name/repo_name` path. However, private package supports only the `org/repo` path. 

You can use private packages by specifying `org/repo` in the `private` key:
```